### DISCUSSION TITLE
Add Vulcan Verify to Farcaster

### DISCUSSION THREAD

**Discussion by MeltedMindz** [farcasterxyz/protocol#146] [FIP Stage 1: Ideas]
A lot of people aren't yet comfortable with connecting their vaults to farcaster which contain many of their assets that will be used for token gated portions of farcaster.

I believe adding the ability to use vulcan verify to prove asset ownership will help with the growth of farcaster. 

[https://www.vulcan.xyz/](url)

**Comment by vrypan** [comment]
Most users Farcaster journey:
1. Sign up with Warpcast -> Warpcast creates a new ethereum address and a private key. Your wallet is not even touched or known. You don't even need to have a wallet to sign up with Warpcast.
2. Verify addresses -> This is a (optional) simple way to prove you own other Eth addresses. You don't have to give permission to Warpcast or Farcaster, just sign a message proving you own the address. This is what most of us did to associate our Etehreum wallets (containing other assets) with Farcaster.

So, if there is actually a concern, the best way to fix this is better communication.
